File: app/src/main/java/com/iita/akilimo/utils/enums/EnumPotatoProduceType.java
package com.iita.akilimo.utils.enums;

public enum EnumPotatoProduceType {
    TUBERS {
        @Override
        public String produce() {
            return "tubers";
        }
    },
    FLOUR {
        @Override
        public String produce() {
            return "flour";
        }
    };
    public abstract String produce();
}

File: app/src/main/java/com/iita/akilimo/utils/enums/EnumPotatoProduceType.kt
package com.iita.akilimo.utils.enums

enum class EnumPotatoProduceType {
    TUBERS {
        override fun produce(): String {
            return "tubers"
        }
    },
    FLOUR {
        override fun produce(): String {
            return "flour"
        }
    };

    abstract fun produce(): String
}
